pkg/model/event.proto

```diff
@@ -31,6 +31,8 @@ message Event {
     // The key/value pairs that are attached to event.
     // This is intended to be used to specify additional attributes of event.
     map<string,string> labels = 5;
+    // A unique identifier consists of its own name and labels.
+    string event_key = 6 [(validate.rules).string.min_len = 1];
```

Comment:
Where is the hash value that we talked about before?

Reply:
My apologies, I totally forgot it.

Reply:
Thank you. I've done added the hash just now.